README.md

```diff
@@ -1,15 +1,15 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
-- Recursion
-- Recursive Case
-- Base Case
-- Activation Chain/Stack
-- Activation Record/Call
-- Infinite Recursion/Stack Overflow/Stack too deep
-- Tail Recursion
+- Recursion - Recursion occurs when a method calls itself.
+- Recursive Case - A recursive case is the part of the method that contains the recursion.
+- Base Case - The base case is the part of the method that stops the recursion (and produces an answer I think).
```

Comment:
re: `(and produces an answer I think)`

I've never thought about it like that, but I do think that's true.